packages/webapp/src/components/Animals/DetailCards/Other.tsx

```diff
@@ -73,11 +73,9 @@ const OtherDetails = ({
         render={({ field: { onChange, value } }) => (
           <ReactSelect
             label={t('ANIMAL.ATTRIBUTE.ORGANIC_STATUS')}
-            optional
-            value={value}
+            value={value ?? organicStatusOptions.find(({ value }) => value === 'Non-Organic')}
```

Comment:
That was super quick 😳, thank you!!!
I think this default value isn't integrated with react-hook-form and needs to be set through `usePopulateDetails`...

Reply:
Oh, thank you! I assumed setting the value in the ReatSelect would update the form value as well, but it was lazy of me not to check 🙇 

It can definitely be done in `usePopulateDetails` but do you prefer that pattern rather than some sort of setting in the component? It's a little off from the other logic in that hook (which is all basics -> details) and I think it would have be replicated in the next container too 🤔

Reply:
Sorry, that was just the first approach that came to mind, and I'm not particular about it!!

Reply:
The other option would be like this, so... it's not that beautiful either really 😂 

```ts
// Other.tsx

  useEffect(() => {
    if (!getValues(`${namePrefix}${DetailsFields.ORGANIC_STATUS}`)) {
      setValue(
        `${namePrefix}${DetailsFields.ORGANIC_STATUS}`,
        organicStatusOptions.find(({ value }) => value === 'Non-Organic'),
      );
    }
  }, []);
  ```

Reply:
But all the changes would be in the `Other.tsx`, right? 😂 
I'm good with either way!

Reply:
Correct, only that change added to `Other.tsx`

@Duncan-Brain can pick when he reviews 😆

Reply:
It seems like hooks are preferred place for storing this initial value logic -- `Other.tsx` looks pretty clean right now!

Reply:
I think the mess added to `Other.tsx` by putting it there is less than the mess added to `usePopulateDetails` 😂  Also I do think it fixes the bug if you could please re-check 🙏 

Thank you for spotting that @Duncan-Brain!!!!